plans/jenkins/jenkins.yaml

```diff
@@ -47,15 +83,122 @@ jenkins:
         vpc_id: !lookup /jenkins-vpc/outputs/vpc_id
         subnet: !lookup /jenkins-vpc/outputs/subnet_id1
         instance_name: "jenkins"
-    - name: jenkins-agent
+
+    - name: jenkins-agent-primary
+      extension: EC2Instance
```

Comment:
Since we are modding this plan; might be a good time to add the required custom tags so they don't get purged by the repear in the AWS account